src/main/java/com/iexec/worker/tee/scone/LasService.java

```diff
@@ -53,9 +53,9 @@ boolean start() {
                 .maxExecutionTime(0)
                 .build();
         if (!imageUri.contains(sconeConfig.getRegistryName())) {
-            // FIXME: throw an IllegalArgumentException
-            throw new RuntimeException(String.format("LAS image (%s) is not " +
-                    "from a known registry (%s)", imageUri, sconeConfig.getRegistryName()));
+            log.error("LAS image ({}) is not from a known registry ({})",
+                    imageUri, sconeConfig.getRegistryName());
```

Comment:
Note for later: we should use the standard notation:
```suggestion
            log.error("LAS image is not from a known registry [image:{}, registry:{}]",
                    imageUri, sconeConfig.getRegistryName());
```

Reply:
Sure, feel free to add the issue to the backlog